manim/mobject/opengl/opengl_vectorized_mobject.py

```diff
@@ -107,6 +107,7 @@ def __init__(
         triangulation_locked: bool = False,
         **kwargs,
     ):
+        self._is_initialized: bool = False
```

Comment:
Perhaps this should be replaced with a computed property, something like

```
@property
def _is_initialized(self):
    return hasattr('something_set_by_init_shader_data', self) and ...
```

But probably this doesn't really make that much of a difference; just wanted to avoid introducing some sort of inconsistency regarding the `_is_initialized` attribute.

Reply:
I attended your fix, i just completely removed it because i found the bug over on 3b1b with the order of things so now it is computed in faster than lightspeed time because we just don't need it anymore!